data_steward/analytics/cdr_ops/notebook_utils.py

```diff
@@ -1,12 +1,82 @@
 """
-Utility functions for notebooks related to CDR generation
+Utility functions for notebooks
 """
+import subprocess
 
-IMPERSONATION_SCOPES = [
-    'https://www.googleapis.com/auth/drive.readonly',
-    'https://www.googleapis.com/auth/bigquery'
+import sqlalchemy
+
+from gcloud.gsm import SecretManager
+from common import CDR_SCOPES
+from utils.auth import get_impersonation_credentials
+
+IMPERSONATION_SCOPES = CDR_SCOPES + [
+    'https://www.googleapis.com/auth/cloud-platform'
 ]
 
+POSTGRES_USER_NAME = "pdr_psql_username"
+POSTGRES_PASSWORD = "pdr_psql_password"
+POSTGRES_PORT = 7005
+DATABASE_NAME = "drc"
+HOST = "localhost"
+PDR_INSTANCE_NAME = "pdr_cloud_sql_read_only_instance"
+
+
+def stop_cloud_sql_proxy(process):
+    process.terminate()
```

Comment:
We may want a doc string stating this is specifically killing the subprocess that starts the cloud sql proxy.  Just so we know what the motivation is for this function.